test/CountHeartbeats.lean

```diff
@@ -0,0 +1,21 @@
+import Mathlib.Tactic.Ring
+
+set_option linter.style.header false
+
+/-- info: Used 7 heartbeats, which is less than the current maximum of 200000. -/
```

Comment:
How flaky is putting an exact number here? See also #18372 :)

Reply:
I was hoping that I could get a very stable number by picking a very trivial result. In some sense, if this number changes, I wouldn't mind knowing explicitly that it changed.